src/features/rewards/panelWelcome/index.tsx

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { getLocale } from '../../../helpers'
+
+import {
+  StyledWrapper,
+  StyledInnerWrapper,
+  StyledHeaderText,
+  StyledBatLogo,
+  StyledTitle,
+  StyledDescText,
+  StyledFooterText,
+  StyledButtonWrapper,
+  StyledTrademark
+} from './style'
+
+import { BatColorIcon } from '../../../components/icons'
+import Button from '../../../components/buttonsIndicators/button'
+
+export type Variant = 'one' | 'two'
+
+export interface Props {
+  id?: string
+  width?: string
+  variant?: Variant
+  optInAction: () => void
+}
+
+export default class PanelWelcome extends React.PureComponent<Props, {}> {
+  get locale () {
```

Comment:
Please add default prop for variant, so that we have define it

Reply:
let's ignore this one until https://github.com/palantir/tslint/issues/4130 is fixed